Scripts/script-ExtractDomain.yml

```diff
@@ -13,10 +25,76 @@ script: |-
           reg = reg.substring(1);
           return str.match(reg);
       }
-
```

Comment:
For regex match is the logic with the `~` prefix still valid at 3.5?

Reply:
It works with backward too